code/style.css

```diff
@@ -1,4 +1,424 @@
 html {
-  background: #0025ff;
+  box-sizing: border-box;
+}
+
+/***Header for MOBILE***/
+
+header {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  border-bottom: 1px solid #000;
+}
+
+header nav {
+  display: none;
+}
+
+/*** Header for TABLET ***/
+
+@media (min-width: 668px) and (max-width: 1024) {
+  header {
+    display: flex;
```

Comment:
There is no need to re-declare `display: flex` since you've already done it for the header on line 8 🤗